src/wlapplication_options.cc

```diff
@@ -121,6 +121,20 @@ void set_config_string(const std::string& section,
 
 constexpr char kFastplaceNameSeparator = '$';
 
+KeyboardShortcut operator+(const KeyboardShortcut& id, const int i) {
+	return static_cast<KeyboardShortcut>(static_cast<uint16_t>(id) + i);
+}
+
+KeyboardShortcut& operator++(KeyboardShortcut& id) {
+	id = id + 1;
+	return id;
+}
+
+uint16_t operator-(const KeyboardShortcut& id, const KeyboardShortcut& base) {
```

Comment:
Operands to + and - operators should be called `lhs` and `rhs` or `a` and `b`

Reply:
OK, but I still think my names better express what this is actually used for.